docs/keywords/README.md

```diff
@@ -0,0 +1,4 @@
+Keyword section headers
+=======================
+
+This directory contains a set of markdown files whose names match keywords used in plugins. The file will be used to create an intruction in the generated sections for each group of plugins that mention the keyword named in the filename.
```

Comment:
instruction

Reply:
Yes, the readme is here to tell people what to do.